app/views/shared/_message_notifier.html.haml

```diff
@@ -0,0 +1,33 @@
+%li
+  %ul.nav.pull-right
+    %li.dropdown.pull-right
+      = link_to "Notices (#{notifier.message_count})",
+        :notifications,
+        class: "dropdown-toggle",
+        data: { toggle: "dropdown" }
+
+      %ul.dropdown-menu
+        - if notifier.order_details_in_dispute?
+          %li
+            = link_to "#{t("message_notifier.order_details_in_dispute")} (#{notifier.order_details_in_dispute.count})",
+              facility_disputed_orders_path(current_facility)
+
+        - if notifier.notifications?
```

Comment:
I think it might be good to try to come up with a different name for how this is used in this interface. This is kind of confusing.